### ISSUE TITLE
to include NIS/LDAP map

### ISSUE THREAD

**Issue by xm1234567** [voxpupuli/puppet-autofs#142]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet:
- Ruby:
- Distribution:
- Module version:

## What are you seeing
Hello, what I am asking is not an issue. I am testing how to add NIS/LDAP maps:
To add the following two lines in /etc/auto.master:
/nethome ldap:nisMapName=auto.nethome,dc=mydom,dc=com
/net yp:auto.net

I use :
include autofs
autofs::mount { 'add_nis_map':
  ensure  => 'present',
  mount   => '/net',
  mapfile => 'yp:auto.net',
}
autofs::mount { 'ldap_map':
  ensure  => 'present',
  mount   => '/nethome',
  mapfile => 'ldap:nisMapName=auto.nethome,dc=mydom,dc=com',
} 

It is working well.

 But if I want to add  just one line  '+auto.master' in /etc/auto.master is not possible. Do you plan to add this possibility in the future or not?

Thanks

**Comment by bastelfreak** [comment]
I don't think that we've specific plans for it. Do you want to provide a PR for it?

**Comment by xm1234567** [comment]
Sorry for the late reply. I am not ready to provide a PR yet :)
I am wondering, maybe just using "mapfile => 'yp:auto.net'," is an alternative way to add specific NIS map, although it is not exactly as '+auto.master'. 

Before I was using this module ( https://forge.puppet.com/pdxcat/autofs ) to add our NIS maps auto.master; which has not been updated for 3 years. So I plan to change to your module. 

Now we are changing from NIS to IDM and LDAP, probably we will use ldap in the future, not use +auto.master. But during my test, I notice this, thinking maybe someone else could have this need. That is why I pose this question.

**Comment by hubizx** [comment]
hi, Almost started using voxpupuli/puppet-autofs, but missing support +auto.master stopped me doing that.

**Comment by xm1234567** [comment]
maybe you can provide an patch, but please make ii as an option, not mandatory in the conf.

**Comment by erobhal** [comment]
Pull request for this feature has been sent.

https://github.com/voxpupuli/puppet-autofs/pull/183
